tuna/miopen/miopen_lib.py

```diff
@@ -310,6 +340,12 @@ def run(self):
     if self.args.add_tables:
       self.add_tables()
       return None
+
+    if self.args.subcommand is not None and self.args.subcommand == 'import_configs':
+      self.set_import_cfg_batches()
+      run_import_configs(self.args, self.logger)
```

Comment:
Could pass self.args.import_configs here.

Reply:
self.args.import_configs does not contain arguments like: config_type, arch, num_cu etc which are part of the shared args, so the whole thing needs to be passed.

Reply:
Note that get_import_cfg_parser does include config_type and that arch and num_cu are not relevant in this context.
config_type would be present at the submodule namespace level 'self.args.import_configs'.
I went through import_configs again to be sure and args.import_configs universally replaces args in this diff with exception to one location referencing config_type, but that is available in both namespaces. So passing self.args.import_configs would remove the need for this indirection.